orchestrators/ProcurementOperations/src/Services/PaymentProcessingService.php

```
@@ -113,622 +77,175 @@ public function createBatch(
             totalAmount: $batch->totalAmount,
-            transactionCount: $batch->itemCount,
-            messageId: $batch->batchId,
-            creationDateTime: new \DateTimeImmutable(),
-            numberOfTransactions: $batch->itemCount,
-            controlSum: $batch->totalAmount->getAmount(),
-            initiatingParty: $batch->tenantId,
-            schemaVersion: 'pain.001.001.03',
-            validationPassed: true,
-            generatedBy: 'system',
+            includedPaymentIds: array_map(fn($item) => $item->paymentItemId, $batch->paymentItems),
         );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\ProcurementOperations\DTOs\Financial\BankFileGenerationResult' in method 'generateCheckPrintFile'. (undefined)

(StaticAccess)